koku/reporting/provider/all/openshift/models.py

```diff
@@ -38,6 +38,8 @@ class Meta:
             GinIndex(fields=["tags"], name="ocpall_tags_idx"),
             models.Index(fields=["product_family"], name="ocpall_product_family_idx"),
             models.Index(fields=["instance_type"], name="ocpall_instance_type_idx"),
+            # A functional index named "ocpall_product_code_ilike" was created manually via RunSQL migration operation
```

Comment:
Would you mind adding similar comments to the other model files that have functional indexes getting created?

Reply:
@adberglund 
Excellent catch! I have added comments.